source/sync/data-model/update-data-model/update-schema-development-apps.txt

```diff
@@ -0,0 +1,199 @@
+  Development Mode is enabled. Refer to :ref:`anonymous-authentication`
+  for more information.
+- Deployment drafts are disabled. You *cannot* re-enable deployment drafts in the
+  UI while Development Mode is enabled. However, you can still manually create
+  deployment drafts through the CLI or the Admin API. Refer to
+  :ref:`deployment-drafts` for more information.
+
+For more information on Development Mode and other Device Sync settings available in the
+App Services UI, refer to :ref:`sync-settings`.
+
+.. include:: /includes/note-dev-mode-production.rst
+
+Data Model Changes in Development Mode
+--------------------------------------
+
+Changes to a Device Sync data model are considered either breaking or
+non-breaking. 
+
+Non-Breaking Changes
+~~~~~~~~~~~~~~~~~~~~
+
+Non-breaking changes are handled the same regardless of whether Development Mode is enabled or disabled. You can make these changes in your server-side App
+Services schema or in your client-side object model, and they apply automatically.
+
+Non-breaking changes in Development Mode include:
+
+- Add an object type
+- Add a default value on a property
+- Add a required property
+- Add an optional property
+
+Breaking Changes
+~~~~~~~~~~~~~~~~
+
+When Development Mode is enabled, you can make breaking changes from
+your client-side object model, and Sync will attempt to apply the updates to the server-side App Services
+schema the next time your app connects.
+
+Breaking changes in Development Mode include:
+
+- Remove an object type *(non-breaking change only if made from the client-side object
+  model)*
+- Remove a property *(non-breaking change only if made from the client-side object
+  model)*
+- Rename an object type
+- Rename a property
+- Change the property type
+- Make a required property optional
+- Make an optional property required
+
+.. tip:: Remap a Name Instead of Renaming
+
+   Several Device SDKs support remapping a property or class name. Depending on
+   your use case, renaming
+   through the client code
+   can be a convenient
+   alternative to renaming your properties or classes. To learn how, refer to the following SDK documentation:
+
+   - C++ SDK - not currently supported
+   - :ref:`Flutter SDK - Remap a Property or Class <flutter-map-property>`
+   - :ref:`Java SDK - Rename a Field or Class <java-rename-field>`
+   - :ref:`Kotlin SDK - Remap a Property or Class <kotlin-remap-a-property>`
+   - :ref:`.NET SDK - Rename a Property or Class <dotnet-rename-property>`
+   - :ref:`Node.js SDK - Remap a Property or Class <node-remap-a-property>`
+   - :ref:`React Native SDK - Remap a Property <react-native-remap-a-property>`
+   - :ref:`Swift SDK - Remap a Property <swift-rename-property>`
+
+Make Breaking Changes from Your Client-Side Object Model
+--------------------------------------------------------
+
+The following describes how to make breaking changes to your client-side object model using Development Mode.
+
+.. Prerequisites
```

Comment:
Apparently the procedure steps' heading level is set by the preceding. I'd prefer them to be H3, but I'm not sure how to handle this section heading